spec/spec.md

```diff
@@ -423,21 +448,22 @@ To create a `did:dht`, the process is as follows:
 
 5. Submit the result of to the [[ref:DHT]] via a [[ref:Pkarr]] relay, or a [[ref:Gateway]], with the identifier created in step 1.
 
-::: note
-This specification **does not** make use of JSON-LD. As such it is prohibited to inclue an `@context` property in the DID Document.
-:::
+This specification **does not** make use of JSON-LD. As such DID DHT Documents ****MUST NOT**** include the `@context` property.
 
 #### Read
 
 To read a `did:dht`, the process is as follows:
 
-1. Take the suffix of the DID, that is, the _[[ref:z-base-32]] encoded identifier key_, and pass it to a [[ref:Pkarr]]
+1. Take the [[ref:suffix]] of the DID, that is, the _[[ref:z-base-32]] encoded identifier key_, and submit it to a [[ref:Pkarr]]
 relay or a [[ref:Gateway]].
 
 2. Decode the resulting [[ref:BEP44]] response's `v` value using [[ref:bencode]].
 
 3. Reverse the DNS [property mapping](#property-mapping) process and re-construct a conformant [[ref:DID Document]].
 
+    a. Expand all identifiers to their fully-qualified form (e.g. `did:dht:uodqi99wuzxsz6yx445zxkp8ddwj9q54ocbcg8yifsqru45x63kj#0` 
```

Comment:
@frankhinek addressing https://github.com/TBD54566975/did-dht-method/issues/130 here

Reply:
when this calls out "all identifiers", just so I understand, isn't only the Identity Key represented as "#0" or "0"? also this refers to verification methods, right? are there other parts of the did doc that could be represented as `#xxx`?

Reply:
I mean to say a DID Doc should always use fully qualified identifiers, regardless of the verification method id

it is confusing to handle `#0` or `#key-1` as opposed to `did:dht:example#0` or `did:dht:example#key-1`

makes it tougher for implementations to handle some cases with fragments and others without